pkg/utils/ptr/ptr.go

```diff
@@ -30,3 +30,7 @@ func Node(node v1.Node) *v1.Node {
 func Quantity(quantity resource.Quantity) *resource.Quantity {
 	return &quantity
 }
+
+func Val[T any](v T) *T {
```

Comment:
```suggestion
func To[T any](v T) *T {
```
Idiomatically:
```
ptr.To(mything)
```

Reply:
Also, this use of generics is awesome and we should replace all other ptr methods w/ it. Maybe we need a corresponding `From`.